CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
+if( NOT SKIP_GOOGLETEST_INSTALL )
+   set( ENABLE_GOOGLETEST_INSTALL ON )
```

Comment:
That's the caveat I mentioned in my previous message; in that case you need `set(FOO ON CACHE "" INTERNAL)`. It still works, as long as the outer project is aware to pre-set `FOO` as a *cache* value.